web-app/src/components/BookingModal/BookingModalForm/container.js

```diff
@@ -170,6 +189,8 @@ const Container = Wrapped =>
           toggleRejectionMessageView={toggleRejectionMessageView}
           capturedRejectionReponseText={capturedRejectionReponseText}
           assignRejectionResponseText={this.assignRejectionResponseText}
+          getApprovedDays={this.getApprovedDays()}
+          getPendingDays={this.getPendingDays()}
```

Comment:
Rename these to "approvedDays" and "pendingDays". The function names are fine but the prop name makes these props sound like functions.